integration/kubernetes/singleMaster-hdfsJournal/master/alluxio-master-statefulset.yaml.template

```diff
@@ -45,6 +45,7 @@ spec:
       nodeSelector:
       securityContext:
         fsGroup: 1000
+      initContainers:
```

Comment:
This is because we moved the if-else to inside the `initContainers`. It's fine to have `initContainers` as empty.